code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -8,7 +8,7 @@ var/global/posibrain_notif_cooldown = 0
 	w_class = 3
 	origin_tech = "biotech=3;programming=3;plasmatech=2"
 	var/notified = 0
-	var/askDelay = 10 * 60 * 1
```

Comment:
**I HONESTLY DO NOT KNOW WHY I DID IT THIS WAY! MAYBE IT WAS HERE WHEN I STOLE IT!**